automerge/src/autocommit.rs

```diff
@@ -298,6 +298,10 @@ impl Transactable for AutoCommit {
         self.doc.length_at(obj, heads)
     }
 
+    fn object_type<O: AsRef<ExId>>(&self, obj: O) -> Result<ObjType, AutomergeError> {
```

Comment:
Should this be `pub` too?

Reply:
This is part of a trait, so public by default.